### PR TITLE
Fix: 1637 Always display a Tutorial in Desktop mode 

### PR DESCRIPTION
**Pull request by Rajat-dhyani**
Currently, if the preview is set to Mobile view and you click to view the tutorial, it remains in the mobile view. This fix always show the Tutorial file in desktop view. This means that if we're previewing the index.html page in the mobile view, we return the preview to mobile after showing the tutorial.
Fixes : [1637](https://github.com/mozilla/thimble.mozilla.org/issues/1637)

### CODE REVIEW

File: public/editor/scripts/editor/js/fc/bramble-ui-bridge.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ define(function(require) {
 
   var adapting = false;
   var adaptTimeoutMS = 200; // How often we adapt editor bar layout
+  var _isMobileView = false; // to check wheather mobile view is on/off
```
Comment (Pomax) [inline public/editor/scripts/editor/js/fc/bramble-ui-bridge.js]: please don't use underscores in variables: there is nothing special about this variable and it deserves a normal name. Also on a typo issue, `wheather` -> `whether`

**Comment by flukeout** [comment]
So just to clarify, a couple of changes are needed here...

* Remove the underscore from the variable
* Fix the "whether" spelling

**Comment by Rajat-dhyani** [comment]
@humphd yes sir i actually took the cue from brackets code base to name the private variable starting with underscore.. @Pomax @flukeout i did the changes accordingly please check this pr.

**Comment by humphd** [comment]
I'm confused about these changes.  It seems to me that we should be making them in Brackets, so that when you open a tutorial, we automatically switch to Desktop mode.  I thought you had a patch happening for that already?  Why this fix @Rajat-dhyani?

**Comment by Rajat-dhyani** [comment]
actually sir @humphd first i did the changes in brackets only but then there i couldn't preserve the mobile state of the preview mode... as is stated in the problem statement that we have to switch back to mobile view in the preview mode... so i did the changes here instead.

**Comment by humphd** [comment]
@Rajat-dhyani just a note to let you know that this is still on my radar, and among the many reviews I'm working through.   I'll try to get through them this week.

**Comment by Rajat-dhyani** [comment]
@humphd sounds great sir.. thanks for letting me know :)

**Comment by humphd** [comment]
OK, sorry for the delay in this.  I was looking for my notes on this, and I suspect that I must have said this in chat vs a bug, so let me repeat it.

What we need to do is fix this inside Brackets, not Thimble. The tutorial override is set here:

https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/Tutorial.js#L20-L36

We need to add some code there to also force us into Desktop mode, which happens here:

https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/UI.js#L225-L248

So you basically need to call `UI.showDesktopView()` when you set the tutorial override.  The reason this will fix Thimble is because we emit events to let it know about the change to tutorial/view modes:

https://github.com/mozilla/brackets/blob/master/src/extensions/default/bramble/lib/UI.js#L243

I'm going to close this PR, and ask that you instead fix it in Brackets according to what I've just written here.  Let me know if you have questions about that, and ideally in the bug or a PR so we don't lose track of the info.

Thanks for working on this.

**Pull request closed without merging**